lib/unhangout-routes.js

```diff
@@ -788,6 +795,136 @@ module.exports = {
         // Starting and stopping events
         //
 
+        app.get("/followup/event/:id/participant_:participantCount", function(req, res) {
```

Comment:
"count" is usually used to refer to a total, whereas "index" is used to refer to a position within an array.  So it'd be preferable to use "participantIndex" here rather than "participantCount".

But more importantly: the userId's are stored in the event history in a non-ordered data structure (they're stored in a javascript object, which has non-deterministic ordering).  As a result, an index to track which user you're looking at is unreliable.  It'd be better to use a userId to choose which user to display.